src/gui/widgets/debugwidget.cpp

```diff
@@ -148,24 +149,32 @@ void DebugWidget::on_importCsvPushButton_clicked()
     if (file.exists() && file.isFile()) {
 
         if (ADataBaseSetup::commitData(aReadCsv(file.absoluteFilePath()), ui->tableComboBox->currentText())) {
-            auto mb = QMessageBox(this);
-            mb.setText("Data inserted successfully.");
-            mb.exec();
+            QMessageBox message_box(this);
+            message_box.setText("Data inserted successfully.");
+            message_box.exec();
         } else {
-            auto mb = QMessageBox(this);
-            mb.setText("Errors have ocurred. Check console for details.");
-            mb.exec();
+            QMessageBox message_box(this);
+            message_box.setText("Errors have ocurred. Check console for details.");
+            message_box.exec();
         }
     } else {
-        auto mb = QMessageBox(this);
-        mb.setText("Please select a valid file.");
-        mb.exec();
+        QMessageBox message_box(this);
+        message_box.setText("Please select a valid file.");
+        message_box.exec();
     }
 }
 
 void DebugWidget::on_debugPushButton_clicked()
 {
-
+#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
+    DEB << "SSL version use for build: " << QSslSocket::sslLibraryBuildVersionString();
+    DEB << "SSL version use for run-time: " << QSslSocket::sslLibraryVersionNumber();
+    DEB << QT_VERSION_MAJOR << QT_VERSION_MINOR << "At least 5.12";
+#else
+    DEB << "SSL version use for build: " << QSslSocket::sslLibraryBuildVersionString();
+    DEB << "SSL version use for run-time: " << QSslSocket::sslLibraryVersionNumber();
+    DEB << QT_VERSION_MAJOR << QT_VERSION_MINOR << "Less than 5.12";
+#endif
```

Comment:
since the first two lines are identical this could be written as
```c++
DEB << "SSL version use for build: " << QSslSocket::sslLibraryBuildVersionString();
DEB << "SSL version use for run-time: " << QSslSocket::sslLibraryVersionNumber();
#if  QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
DEB << QT_VERSION_MAJOR << QT_VERSION_MINOR << "At least 5.12";
#else
DEB << QT_VERSION_MAJOR << QT_VERSION_MINOR << "Less than 5.12";
#endif
```

Reply:
actually I forgot to clean that, it was for debugging an issue I had. Removing.
Thanks for the feedback!